### DISCUSSION TITLE
Cursor caret displayed in clause text

### DISCUSSION THREAD

**Discussion by sanketshevkar** [accordproject/web-components#246] [Q&A] [answered]
Since the clause text except variables are supposed to be non-editable, why is the placement of cursor caret allowed there? It feels as if that text is also editable.

**Comment by jolanglinais** [comment, marked as answer]
@Michael-Grover might have a better idea of this.

**Reply by Michael-Grover** [reply]
![image](https://user-images.githubusercontent.com/40569793/107699391-075dc200-6c84-11eb-9e57-71e45f8b05b8.png)

For anyone following this discussion who is not familiar with the term "cursor caret", here's a picture. The vertical line in the word "deeme|d" is a caret that indicates where text will appear when the user types.

@sanketshevkar I agree that the caret should not flash on read only text. Feel free to create an issue for this.

**Reply by sanketshevkar** [reply]
Sure will do that.
